docs/onion-message-channels.md

```diff
@@ -0,0 +1,220 @@
+# HOW TO SETUP ONION MESSAGE CHANNELS IN JOINMARKET
+
+### Contents
+
+1. [Overview](#overview)
+
+2. [Testing, configuring for signet](#testing)
+
+4. [Directory nodes](#directory)
+
+<a name="overview" />
+
+## Overview
+
+This is a new way for Joinmarket bots to communicate, namely by serving and connecting to Tor onion services. This does not
+introduce any new requirements to your Joinmarket installation, technically, because the use of Payjoin already required the need
+to service such onion services, and connecting to IRC used a SOCKS5 proxy (by default, and used by almost all users) over Tor to
```

Comment:
IRC messaging does not use SOCKS5 and .onion with our current default config, so it's not by default.

Reply:
Yes (although, heh, why haven't we fixed that!?). Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/1182/commits/1b62d9c3c13a2a4d9526e0ee8305fb084e360128